csharp/Managed/Patch.cs

```diff
@@ -0,0 +1,44 @@
+using LibPDBinding.Native;
+using System;
+
+namespace LibPDBinding.Managed
+{
+	/// <summary>
+	/// Pd Patch.
+	/// </summary>
+	public class Patch : IDisposable
+	{
+		readonly IntPtr _handle;
+
+		internal Patch (IntPtr handle)
+		{
+			_handle = handle;
+			DollarZero = PInvoke.getdollarzero (_handle);
+		}
+
+		~Patch ()
+		{
+			Dispose (false);
+		}
+
+		public void Dispose ()
+		{
+			Dispose (true);
+			GC.SuppressFinalize (this);
+		}
+
+		private void Dispose (bool disposing)
+		{
+			if (disposing) {
+				PInvoke.closefile (_handle);
```

Comment:
Sorry, this is wrong: `_handle` is not a managed resource, so no `if (disposing)` would be correct.